### PR TITLE
Notifications controller

### PR DESCRIPTION
**Pull request by Darklol**
Добавлен контроллер для получения уведомлений на фронте

### CODE REVIEW

File: src/main/java/org/itmo/eventapp/main/model/dto/TaskNotificationDTO.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.itmo.eventapp.main.model.dto;
+
+import java.io.Serializable;
+
+public record TaskNotificationDTO(
```
Comment (IP-13) [inline src/main/java/org/itmo/eventapp/main/model/dto/TaskNotificationDTO.java:5]: Давай Request/Response называть, уже начали так

File: src/test/resources/db/test-migration/V0_2__create_tables_test.sql [hunk 2]
```diff
@@ -58,7 +58,7 @@ create table if not exists notification(
     title varchar(256) not null,
     description text not null ,
     seen boolean not null default FALSE,
-    read_time timestamp default null
+    sent_time timestamp default null
```
Comment (IP-13) [inline src/test/resources/db/test-migration/V0_2__create_tables_test.sql:61]: А в основной не надо менять? Или там уже так

Comment (Darklol) [reply src/test/resources/db/test-migration/V0_2__create_tables_test.sql:61]: Забыл, сейчас поправлю

**Comment by IP-13** [comment]
|Overall Project|41.09% **`-4.53%`**|:x:|
|:-|:-|:-:|
|Files changed|40.86%|:x:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[NotificationService.java](https://github.com/itmo-events-app/event-app/blob/173119e0c24411364b459d733d00e821675a8fe0/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FNotificationService.java)|85.83% **`-9.45%`**|:green_apple:|
|[EventRoleService.java](https://github.com/itmo-events-app/event-app/blob/173119e0c24411364b459d733d00e821675a8fe0/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FEventRoleService.java)|14.34% **`-9.43%`**|:x:|
|[NotificationController.java](https://github.com/itmo-events-app/event-app/blob/173119e0c24411364b459d733d00e821675a8fe0/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FNotificationController.java)|7.59% **`-92.41%`**|:x:|

**Review by IP-13 — APPROVED** [review]

**Pull request merged**